### DISCUSSION TITLE
Search Widget: Implement Fuzzy Search With Fuse.js

### DISCUSSION THREAD

**Discussion by tiago-sartor** [elementor/elementor#28210] [Feature Request]
### Prerequisites

- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.
- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.

### What problem is your feature request going to solve? Please describe..

The new search widget is awesome and almost completely solves a big headache which was relying on 3rd party plugins for better search. However, the Search Widget could become even better with a fuzzy search functionality.

### Describe the solution you'd like.

Fuse.js (https://www.fusejs.io/) is a lightweight fuzzy search engine that could be implemented to improve Elementor's Search Widget.

### Describe alternatives you've considered.

Only 3rd party plugins (paid ones) offer fuzzy search.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by jazir555** [comment]
ElasticSearch + ElasticPress will let you do Fuzzy Search for free, but you'll need to be able to install the free server level ElasticSearch service to use the ElasticPress for free. If you're on shared hosting you likely won't be able to do so.
